src/net/sourceforge/kolmafia/persistence/DebugDatabase.java

```diff
@@ -908,9 +907,8 @@ private static void checkEquipmentMap(final PrintStream report, ItemMap imap) {
     report.println("# " + tag + " section of equipment.txt");
     report.println();
 
-    Object[] keys = map.keySet().toArray();
-    for (Object key : keys) {
-      String name = (String) key;
+    String[] keys = map.keySet().toArray(new String[0]);
+    for (String name : keys) {
```

Comment:
Can we not iterate the keyset here too?

Reply:
wtf; did I really forget to continue past the first one??
o///o